src/lib/math.js

```diff
@@ -18,8 +18,40 @@ const $builtinmodule = function (name) {
         return new Sk.builtin.float_(Math.ceil(_x));
     };
 
-    function comb(x, y) {
-        throw new Sk.builtin.NotImplementedError("math.comb() is not yet implemented in Skulpt");
+    function comb(n, k) {
+        n = Sk.misceval.asIndexOrThrow(n);
+        k = Sk.misceval.asIndexOrThrow(k);
+        if (n < 0) {
+            throw new Sk.builtin.ValueError("n must be an non-negative integer");
+        }
+        if (k < 0) {
+            throw new Sk.builtin.ValueError("k must be a non-negative integer");
+        }
+        if (k > n) {
+            return new Sk.builtin.int_(0);
+        }
+        n = new Sk.builtin.int_(n);
+        k = new Sk.builtin.int_(k);
+        let temp = n.nb$subtract(k);
+        if (Sk.ffi.remapToJs(temp) < Sk.ffi.remapToJs(k)) {
```

Comment:
you've got a lot of `remapToJs` going on. It's worth noting that `remapToJs` is a slow operation. So doing it several times on the same object is unnecessary.